### PR TITLE
Set up Cloud Build to run tests.

### PR DESCRIPTION
**Pull request by dseomn**

### CODE REVIEW

**Comment by dseomn** [comment]
I'm not able to set up the Cloud Build GitHub app yet, but this PR is sufficient for Cloud Build to run automated tests on branches, without pull request integration. So for now, we'll have to manually check Cloud Build before merging.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Dockerfile for running tests only.
+
+
+FROM python:3.6-slim
```
Comment (dcbrandao) [inline Dockerfile]: I believe we should be moving toward python3.7

Comment (dseomn) [reply Dockerfile]: There's still a 3.6 environment that I want to use this library in, and so far there aren't any 3.7 features I want to use in the library. Do you think it's worth running the tests in both versions?

Comment (dseomn) [reply Dockerfile]: As discussed offline, I'll merge this as-is.

**Pull request merged**